---
string performance
float32 current_time
bool running
bool paused
